tests/test_tictactoe.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+import numpy as np
+import random
+from metagamer.environments import tictactoe
+
+
+@pytest.fixture
+def tt():
+    return tictactoe.TicTacToeEnv()
+
+
+def test_simple_game(tt):
+    _, reward, done, _ = tt.step((0, 0))
+    assert not done
+    assert reward == 0
+    _, reward, done, _ = tt.step((0, 1))
+    assert not done
+    assert reward == 0
+    _, reward, done, _ = tt.step((1, 0))
+    assert not done
+    assert reward == 0
+    _, reward, done, _ = tt.step((1, 1))
+    assert not done
+    assert reward == 0
+
+    _, reward, done, _ = tt.step((2, 0))
+    assert done
+    assert reward == 1
```

Comment:
Nice, this helps me see how the game would be evolved.

Reply:
I'll add some comments to make this clearer (as I'm confused by this)

But yeah, this is a good example of how tests can make code more understandable.

Reply:
Added a bit more doc